pinot-hadoop/pom.xml

```diff
@@ -103,6 +103,28 @@
       <version>${project.version}</version>
       <classifier>shaded</classifier>
     </dependency>
+    <dependency>
+      <groupId>org.apache.pinot</groupId>
+      <artifactId>pinot-parquet</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.pinot</groupId>
+          <artifactId>pinot-core</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.pinot</groupId>
```

Comment:
are you trying to shade the orc and parquet dependencies? this pr only shades the google dependency, right?

Reply:
This pr aims to shade orc/parquet, so user could use this jar to create segment from parquent/orc file. Otherwise they need to drop pinot-parquet/pinot-orc jar into their classpath or do their own shading with pinot-parquet/pinot-orc to generate a fat jar.